src/components/ProgressForm/ProgressForm.tsx

```diff
@@ -0,0 +1,79 @@
+import NavBar from '../navBar';
+import InputWithQuestions from './InputWithQuestions';
+import ProgressHeader from './ProgressHeader';
+import styles from '@/components/ProgressForm/ProgressForm.module.scss';
+
+const questions = [
+    {
+        id: 1,
+        name: 'progress',
+        question: 'Task Progress after the previous update',
+    },
+    {
+        id: 2,
+        name: 'plan',
+        question: 'Planned progress before the next update',
+    },
+    {
+        id: 3,
+        name: 'blockers',
+        question: 'List down any blockers that you have',
+    },
+];
+
+const getFormattedDate = (): string => {
+    const date = new Date();
+    const month = [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December',
+    ];
+
+    const daysInWeek = [
+        'Sunday',
+        'Monday',
+        'Tuesday',
+        'Wednesday',
+        'Thursday',
+        'Friday',
+        'Saturday',
+    ];
+
+    return `${daysInWeek[date.getDay()]}, ${date.getDate()} ${
+        month[date.getMonth()]
+    } ${date.getFullYear()}`;
+};
+
+const ProgressForm = () => {
+    return (
+        <>
+            <NavBar />
+            <ProgressHeader/>
```

Comment:
as the name of this component suggests it should only have the form, remove these
```
  <NavBar />
            <ProgressHeader/>
```

Reply:
Do you want me to create a layout for it?

Reply:
Created a layout